tests/e2e/specs/dashboard/dashboard-noscript.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Site Kit noscript notice', () => {
+	beforeEach( async () => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-splash' );
+	} );
+
+	describe( 'When Javascript is enabled', () => {
+		it( 'Should not display noscript notice', async () => {
+			const noscript = await page.$( '#wpbody-content' );
+			await expect( noscript ).not.toMatchElement( '.googlesitekit-noscript' );
```

Comment:
No reason to not combine the selector here. Also using `JSHandle`s such as `noscript` is here requires that they are disposed of when you do save a reference.

This should simply be:
```js
await expect( page ).not.toMatchElement( '.googlesitekit-noscript', { visible: true } );
```
Let's include `visible` as well to be explicit, since the element should always be on the page, it just shouldn't be seen unless JS is disabled.